addons/test_mail/tests/test_message_track.py

```diff
@@ -346,8 +346,8 @@ def test_track_groups(self):
 
         msg_emp = self.record.message_ids.message_format()
         msg_sudo = self.record.sudo().message_ids.message_format()
-        self.assertFalse(msg_emp[0].get('tracking_value_ids'), "should not have protected tracking values")
-        self.assertTrue(msg_sudo[0].get('tracking_value_ids'), "should have protected tracking values")
+        self.assertEqual(msg_emp[0].get('trackingValues'), [('insert-and-replace', [])], "should not have protected tracking values")
+        self.assertTrue(msg_sudo[0].get('trackingValues'), "should have protected tracking values")
```

Comment:
This assert is no longer clear enough, I would expect an assertEqual like above and see the shape of the result. This would be True even if the result was empty.